activerecord/lib/active_record/insert_all.rb

```diff
@@ -57,7 +57,6 @@ def primary_keys
       Array(connection.schema_cache.primary_keys(model.table_name))
     end
 
-
```

Comment:
Previously Basecamp guys said these newlines were intentional for method grouping.

https://github.com/rails/rails/pull/38899#discussion_r414679692